packages/sui-mock/README.md

```diff
@@ -0,0 +1,135 @@
+# sui-mock
+
+### Installation
+
+```sh
+npm install @s-ui/mock --save-dev
+```
+
+### Mockers
+
+#### **Request mocking with `setupMocker`**
+
+It returns all methods included in `setupWorker` and `setupServer`. It will work in browser and server sides.
+
+👉 Check `setupWorker` and `setupServer` in [MSW docs](https://mswjs.io/docs/api/).
+
+
+### Usage
+
+#### 1. Create Request handlers
+Create a `./mocks` folder in your project root and create a `index.js` file inside it.
+
+This index.js should export an array of [Request handlers](https://mswjs.io/docs/basics/request-handler).
+
+**Example:**
+
+Given, a provider endpoint handler
+
+```js
+// ./mocks/exampleGateway/user/handlers.js
+import {rest} from 'msw'
+import {apiUrl} from '../config.js'
+
+const responseResolver = () => res(ctx.status(200), ctx.json({name: 'John Doe'}))
+
+export const getUserHandler = rest.get(`${apiUrl}/user`, responseResolver)
+```
+
+Then, we export a list of handlers
+
+```js
+// ./mocks/handlers.js
+import {getUserHandler} from './mocks/exampleGateway/user/handlers.js'
+
+export default [getUserHandler]
+```
+
+#### 2. Expose mocker from mocks folder
+
+We should use browser or server getMocker to create a mocker instance ready to use in our app. See how it will looks like:
```

Comment:
Once we have the handlers created, we will need to create a mocker with the handlers already defined.